external/concealer/src/execute.cpp

```diff
@@ -45,4 +51,28 @@ int execute(const std::vector<std::string> & f_xs)
 
   return ::execvp(argv[0], argv.data());
 }
+
+void sudokill(pid_t process_id)
+{
+    char *process_str;
+    int status;
+    pid_t pid;
+
+    if (asprintf(&process_str, "%d", process_id) < 0)
+        err(1, "asprintf() failed");
```

Comment:
Something more verbose? 
`asprintf() failed inside sudokill() while trying to terminate Autoware`?